tv_shows_app/src/fetchers/mutators.ts

```diff
@@ -54,3 +54,74 @@ export async function getMutator<T>(url: string, { arg }: { arg: T }) {
   }
   return await response.json();
 }
+
+export async function createReviewMutator<T>(url: string, { arg }: { arg: T }) {
```

Comment:
Pričali smo oko strukture fetcher-a (mutatora) pa smo rekli da mozemo ovo ljepse napisati sve.
Da ne idem u detalje baci pogled kako smo to rješili na predavanju.
Ako budes trebao pomoc mozemo skupa